test/cluster/test_long_query_timeout_erm.py

```diff
@@ -18,10 +18,8 @@
 @skip_mode('release', 'error injections are not supported in release mode')
 @pytest.mark.asyncio
 @pytest.mark.parametrize("query_type,should_wait_for_timeout,shutdown_nodes", [
-    ("SELECT", True, False),
     ("SELECT", True, True),
     ("SELECT_COUNT", False, False),
-    ("SELECT_WHERE", True, False),
```

Comment:
Please, write more elaborated explanation why the removed parametrization is "near duplicates"